src/legacy/core_plugins/region_map/public/region_map_fn.test.js

```diff
@@ -17,7 +17,8 @@
  * under the License.
  */
 
-import { functionWrapper } from '../../interpreter/test_helpers';
+// eslint-disable-next-line
+import { functionWrapper } from '../../../../plugins/expressions/public/functions/tests/utils';
```

Comment:
Why are all of these `functionWrapper` imports disabled?

Reply:
It disables the ESLint rule that does not allow to import deeply from within the plugins. I believe this should be addressed by the owners of the plugins who use the `functionWrapper`. They should probably just copy-paste that function into their tests or even not use it at all. For now I just change the import to import the `functionWrapper` utility from its new home.